docs/reference/commandline/events.md

```diff
@@ -30,7 +30,7 @@ Options:
 
 Docker containers report the following events:
 
-    attach, commit, copy, create, destroy, detach, die, exec_create, exec_detach, exec_start, export, health_status, kill, oom, pause, rename, resize, restart, start, stop, top, unpause, update
+    archive-path, attach, checkpoint, commit, copy, create, destroy, detach, die, exec_create, exec_detach, exec_start, export, extract-to-dir, health_status, kill, oom, pause, rename, resize, restart, start, stop, top, unpause, update
```

Comment:
When the changes to `docker cp` were made a year-and-a-half ago, there was an existing API endpoint named `copy` which emitted the `copy` event. Since the changes added two new API endpoints, I added these two new events so that the new API endpoints still produced their own events. Sorry, I didn't realize at the time that they violated the naming convention (or that there even was one).